module/certificateTemplates/helper.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * name : helper.js
+ * author : Vishnu
+ * created-date : 29-sep-2022
+ * Description : Certificate Template related helper functionality.
+*/
+// dependencies
+const filesHelpers = require(ROOT_PATH+"/module/cloud-services/files/helper");
+const request = require("request");
+
+const { ObjectId } = require("mongodb");
+
+/**
+    * CertificateTemplatesHelper
+    * @class
+*/
+module.exports = class CertificateTemplatesHelper {
+
+  /**
+   * Create certificate template.
+   * @method create
+   * @name create
+   * @param {Object} data - certificate template creation data.
+   * @returns {JSON} created certificate template details. 
+   */
+  
+  static create(data) {
+    return new Promise(async (resolve, reject) => {
+        try {
+            data.issuer.kid = process.env.CERTIFICATE_ISSUER_KID;
```

Comment:
@VISHNUDAS-tunerlabs why do we have to store kid in DB, always we should get it from env when needed.